src/main/kotlin/au/com/timmutton/redexplugin/RedexExtension.kt

```diff
@@ -7,14 +7,20 @@ import java.io.File
  * @author timmutton
  */
 open class RedexExtension(appExtension: AppExtension) {
```

Comment:
the point of having these as files was to be consistent with how store files are used in build types, with the eventual goal being to have redex configurable per build type and/or product flavour. this will require a change on how clients access them, so instead of `configFile = "blah.txt"` it would now be `configFile = file("blah.txt")`

Reply:
Oooh. Okay, I see what you're doing now, I'll switch it back and update the Readme, etc.